### PR TITLE
Add Source Org

### PR DESCRIPTION
**Pull request by ryphil**

### CODE REVIEW

File: content/departments/engineering/teams/Source/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Source Department
+
+The Source department consists of a related set of teams who manage nearly all aspects of interfacing with code hosts. These teams will grow in time, but are small and scrappy, with some in maintenance-mode for now. Source is taking on tremendous challenges in the coming year, expanding our native support beyond Git for huge industry customers still on Perforce, SVN, and in some cases even CVS.
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/Source/index.md]: Maybe I just understood it wrong, but IMHO it sounded a bit like "perforce, SVN and _even_ CVS are so old, but these companies still use them" :)

```suggestion
The Source department consists of a related set of teams who manage nearly all aspects of interfacing with code hosts. These teams will grow in time, but are small and scrappy, with some in maintenance-mode for now. Source is taking on tremendous challenges in the coming year, expanding our native support beyond Git for huge industry customers on Perforce, SVN and CVS.
```

File: content/departments/engineering/teams/Source/strategy.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Source Department Strategy
+
+TL;DR: The Source department builds value for Sourcegrpah by ensuring customers can index all of their code (regardless of code host). We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/Source/strategy.md]: probably parens can be dropped

```suggestion
TL;DR: The Source department builds value for Sourcegrpah by ensuring customers can index all of their code regardless of code host. We believe this is critical for building a long-term defensible value for Sourcegraph as all other code-search solutions are focused on searching across and individual repo or code host.
```

**Review by sashaostrikov — APPROVED** [review]
LGTM!
Left a couple of suggestions.

(It appears that some links are broken)
